tests/run_notebook.py

```diff
@@ -0,0 +1,48 @@
+import pathlib
+
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor
+from nbformat import read
+
+
+def execute_notebook(notebook_path) -> None:
+    notebook = read(notebook_path, as_version=4)
+    execute_preprocessor = ExecutePreprocessor(timeout=600)  # Set the timeout as needed
+
+    execute_preprocessor.preprocess(notebook, {"metadata": {"path": "."}})
+
+
+def update_notebook_interactive_params(notebook_path, take_screenshot: bool = False) -> None:
+    print("Taking screenshot: ", str(take_screenshot))
+
+    # Load the Jupyter notebook
+    with open(notebook_path, encoding="utf-8") as notebook_file:
+        notebook_content = nbformat.read(notebook_file, as_version=4)
+
+    # Iterate through the cells in the notebook
+    for cell in notebook_content.cells:
+        if cell.cell_type == "code":
+            # Check if the code cell contains "Interactive("
```

Comment:
Thanks for the new fixes. Writing also here what we discussed the other day to keep the record. Please move the inside of `if cell.cell_type == "code":` inside a separate function and add some tests for it, I believe that the current `cell.source.replace()` would fail on real world strings, and those tests and having some tests for it will improve development and code mainteinance.